pkg/utils/dangerous_secret_names.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+var DANGEROUS_SECRET_NAMES = []string{
+	"PROMPT_COMMAND",
+	"LD_PRELOAD",
+	"LD_LIBRARY_PATH",
+	"WINDIR",
+	"USERPROFILE",
+	"DYLD_INSERT_LIBRARIES",
+}
+
+type DangerousSecretNameError struct {
+	dangerousNames []string
+	currentConfig  string
+}
+
+func (e *DangerousSecretNameError) Error() string {
+	return fmt.Sprintf("the following dangerous secret names have been found in %s config:\n%s", e.currentConfig, strings.Join(e.dangerousNames, "\n"))
+}
```

Comment:
I think we can avoid needing to print the current config names and therefore needing to add it to the error:

`your config contains the following potentially dangerous secret names:`